cirq/google/engine/engine.py

```diff
@@ -347,7 +349,9 @@ def run_batch(
             for a circuit are listed in the order imposed by the associated
             parameter sweep.
         """
-        if not params_list or len(programs) != len(params_list):
+        if params_list is None:
+            params_list = [dict()] * len(programs)
```

Comment:
Can do `[None] * len(programs)` since `cirq.Sweepable` includes `None` (via [`ParamResolverOrSimilarType`](https://github.com/quantumlib/Cirq/blob/master/cirq/study/resolver.py#L33)).

Reply:
done